app/models/greeting.rb

```diff
@@ -0,0 +1,5 @@
+class Greeting < ApplicationRecord
```

Comment:
- [ ] The app is not running as expected because of cors, kindly make sure to install and setup cors in your rails API so the random messages show as expected